Sources/CoreAPI/WordPressComRestApiErrorDomain.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+/// Error domain of `NSError` instances that are converted from `WordPressComRestApiEndpointError`
+/// and `WordPressAPIError<WordPressComRestApiEndpointError>` instances.
+///
+/// See `extension WordPressComRestApiEndpointError: CustomNSError` for context.
+let WordPressComRestApiErrorDomain = "WordPressKit.WordPressComRestApiError"
+
+// WordPressComRestApiErrorDomain is accessible only in Swift and, since it's a global, cannot be made @objc.
+// As a workaround, here is a builder to init NSError with that domain and a method to check the domain.
```

Comment:
An alternative could be defining the same constant again in Objective-C and making it unavailable to Swift. That way you won't need to change those Objective-C files.

Reply:
See 6fba3a19460d392576003415dd8d54576b16ad21